### ISSUE TITLE
Optionally don't expose schema?

### ISSUE THREAD

**Issue by molexx** [introproventures/graphql-jpa-query#237]
Do all clients need to inspect the schema at runtime?

If not would it be possible to optionally not expose the schema? If so I'd probably want to do that in production.

**Comment by igdianov** [comment]
@molexx It makes sense. I can add configuration property to `GraphQLJpExecutor` to apply `NoIntrospectionGraphqlFieldVisibility` to the schema if enabled.

**Comment by igdianov** [comment]
@molexx This should be possible to achieve with `NoIntrospectionGraphqlFieldVisibility` implementation that already exists in Graphql Java:

https://github.com/graphql-java/graphql-java/blob/master/src/main/java/graphql/schema/visibility/NoIntrospectionGraphqlFieldVisibility.java#L17

After fixing GH-239, it should be possible to apply `NoIntrospectionGraphqlFieldVisibility`, via Supplier implementation.

**Comment by molexx** [comment]
Yes, tested with #239 and NO_INTROSPECTION_FIELD_VISIBILITY following the example with a `@RequestScope` `Supplier<GraphqlFieldVisibility`. Thanks!
